app/css/animator.css

```diff
@@ -505,4 +499,91 @@ a {
 
 ::-webkit-scrollbar-corner       { background-color: #000; }
 
-::-webkit-resizer                { }
\ No newline at end of file
+::-webkit-resizer                { }
+
+/* ======== Preference Windows ======== */
```

Comment:
Eh, the sidebar has been drastically reduced and will continue to be cut... I think it'll be fine to be copied over into index.css.